AmplifyPlugins/API/AWSAPICategoryPlugin/AWSAPIPlugin+Resettable.swift

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2018-2019 Amazon.com,
+// Copyright 2018-2020 Amazon.com,
```

Comment:
I would do this in a separate PR, to reduce the size of this API PR, and keep the discussion focused to API changes.